consensus/propagation/catchup.go

```diff
@@ -22,11 +22,6 @@ func (blockProp *Reactor) retryWants(currentHeight int64) {
 	for _, prop := range data {
 		height, round := prop.compactBlock.Proposal.Height, prop.compactBlock.Proposal.Round
 
-		// don't re-request parts for any round on the current height
```

Comment:
this enables catching up for the current height. However, this wouldn't happen until we're trying to catchup and the network has most likely advanced. This is because we `reset` the timer on every height, so as long as we're receiving blocks and committing them, this shouldn't be called.